app/assets/stylesheets/reskin/moneyhelper_styling_rad.scss

```diff
@@ -0,0 +1,224 @@
+                background-color: #F0F05A;
+                border-color: #8200D1;
+                color: #8200D1;
+                outline: none;
+            }
+            @media print{
+                display: none;
+            }
+            span{
+                display: block;
+                margin-top: -2px;
+            }
+        }
+        .popup-tip__container{
+            border: 2px solid $blue_primary;
+            .popup-tip__close{
+                border: 2px solid $blue_primary;
+                svg{
+                    display: none;
+                }
+                &:before{
+                    content: '';
+                    background-image: $close_svg;
+                    width: 24px;
+                    height: 24px;
+                    background-size: 18px;
+                    background-position: 50%;
+                    background-repeat: no-repeat;
+                    display: block;
+                }
+            }
+        }
+    }
+    .form-section{
+        border-top: 1px solid $support_outline;
+    }
+    .form-section:last-child{
+        border-bottom: 1px solid $support_outline;
+    }
+    select[id^=office_opening_time]{
+        width: auto;
+    }
+    .validation-summary__content-container{
+        border-left: 4px solid $red_error;
+        background-color: $red_error_background;
+        color: $red_error;
+        font-family: 'Roobert-Regular', sans-serif;
+        .validation-summary__title{
+            color: $red_error;
+        }
+    }
+    a:not(.button):not(.navigation__link){
+        color: $magenta_primary!important;
+        text-decoration: none;
+    }
+    .form__row{
+        &.is-errored{
+            border-width: 4px;
+        }
+    }
+    #error_explanation{
+        border-left: 4px solid $red_error;
+        padding-left: 10px;
+        + .registration__row{
+            border-left: 4px solid $red_error;
+            margin: -24px 0 20px 0;
+            .is-errored{
+                border: none;
+            }
+        }
+        h2{
+            color: $text_secondary;
+        }
+        li{
+            color: $red_error;
+            padding-bottom: 10px;
+        }
+    }
+    fieldset{
+        border: none;
+    }
+    .l-questionnaire__legend{
+        font-family: 'Roobert-SemiBold', sans-serif;
+        color: $text_secondary;
+    }
+    .intro{
+        color: $text_secondary;
+    }
+    .validation-summary__list .validation-summary__error a{
+        color: $red_error!important;
+    }
+    .button.button--small.t-remove-button{
+        padding: 8px 16px;
+    }
+    .l-header{
+        background-color: $blue_primary;
+        .mas-logo__link{
+            img{
+                display: none;
+            }
+            background-image: $moneyHelper_svg;
+            background-size: 90px;
+            display: block;
+            width: 90px;
+            height: 50px;
+            background-repeat: no-repeat;
+        }
+    }
```

Comment:
We need to add a style for the `l-header__content` element to deal with the background colour on smaller viewports. This originates in Yeast but will need to be over-written here.

Reply:
There is now a card for this work here: [TP-12719](https://maps.tpondemand.com/entity/12719-rad-self-service-header-background-colour)